File: Odd-Even String Sort.py
def sort_my_string(s):
    odd_s = [j for i,j in enumerate(s) if i%2==1]
    even_s = [j for i,j in enumerate(s) if i%2==0]
    return "".join(even_s)+" "+"".join(odd_s)

File: Odd-Even String Sort.ts
export function sortMyString(S:string):string {
    let odd_s = S.split("").filter((v,i) => i%2==1)
    let even_s = S.split("").filter((v,i) => i%2==0)
    return `${even_s.join("")} ${odd_s.join("")}`
}
